tracing/src/lib.rs

```diff
@@ -192,8 +192,8 @@
 //!
 //! ## Using the Macros
 //!
-//! The [`span!`] and [`event!`] macros use fairly similar syntax, with some
-//! exceptions.
+//! The [`span!`] and [`event!`] macros as well as the `#[tracing::instrument]` attribute
+//! use fairly similar syntax, with some exceptions.
```

Comment:
if you're up for it, it would be nice to make this part of the change on the `master` branch as well --- these docs should be about the same, unlike the attribute's docs.